README.md

```diff
@@ -1,5 +1,10 @@
-Bangle.js App Loader (and Apps)
-================================
+Bangle.js App Loader (and Apps) (FORK - not actively maintained)
```

Comment:
We probably don't want to commit this ;)

Reply:
ahh - just saw you closed it :)

Reply:
Sorry, that was just me being stupid. ... 🤦